### ISSUE TITLE
Make an EnergyPlus Measure that strips out unwanted air temperature and relative humidity added by OpenStudio CLI

### ISSUE THREAD

**Issue by chriswmackey** [ladybug-tools-in2/energy-model-measure#75]
Even though we don't request them at all in the measure, it seems that every OpenStudio model that is created through the CLI has the following extra 4 outputs in it:
![image](https://user-images.githubusercontent.com/5567574/75216745-05dd7600-5763-11ea-9383-be53f20e7033.png)

The requesting of these variables is breaking our usual workflows that request and parse these types of results. I've been at it for the last hour and I can't determine why these values are written in. I don't get them in honeybee-legacy workflows where I just use the C# bindings to forward translate the model from OpenStudio to EnergyPlus.

**Comment by chriswmackey** [comment]
I just wanted to make a note of @DavidGoldwasser 's suggestion here, which is to write an energyplus measure to strip out these extra outputs that the CLI is adding.

**Comment by chriswmackey** [comment]
I found a more elegant solution to this issue.

I realized that it's only the in.idf spit out by OpenStudio CLI contains a lot of extra outputs and the pre-preprocess.idf is just the pure output from running the energy-model-measure (without the extra Zone Air Relative Humidity, etc.). So I have changed the honeybee code that executes the simulation to use the pre-preprocess.idf instead of the in.idf:
https://github.com/ladybug-tools/honeybee-energy/pull/108

This solution means that running the OpenStudio CLI all of the way through the E+ simulation in order to include reporting measures is still going to suffer from the problems I experienced here. But I think that we can deal with this later and hopefully most of the people running reporting measures expect to visualize the simulation outputs with those measures and not the honeybee components that are broken by the auto-dumping of all these extra outputs.
